client/lib/plans/constants.js

```diff
@@ -124,11 +142,30 @@ export const PLANS_LIST = {
 			FEATURE_BASIC_DESIGN,
 			FEATURE_3GB_STORAGE
 		],
-		getBillingTimeFrame: () => i18n.translate( 'for life' )
+		getSignupFeatures: () => [ // pay attention to ordering, it is used on /plan page
+			FEATURE_COMMUNITY_SUPPORT,
+			FEATURE_WP_SUBDOMAIN_SIGNUP,
+			FEATURE_FREE_THEMES_SIGNUP,
+		],
+		getBlogSignupFeatures: () => [ // pay attention to ordering, it is used on /plan page
+			FEATURE_COMMUNITY_SUPPORT,
+			FEATURE_WP_SUBDOMAIN_SIGNUP,
+			FEATURE_FREE_THEMES_SIGNUP,
+		],
+		getPortfolioSignupFeatures: () => [ // pay attention to ordering, it is used on /plan page
+			FEATURE_COMMUNITY_SUPPORT,
+			FEATURE_WP_SUBDOMAIN_SIGNUP,
+			FEATURE_FREE_THEMES_SIGNUP,
+		],
+		getBillingTimeFrame: () => i18n.translate( 'for life' ),
+		getSignupBillingTimeFrame: () => i18n.translate( ' for life' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'for life' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plans-copy-test)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).